themes/src/main/resources/theme/base/login/resources/js/authChecker.js

```diff
@@ -0,0 +1,76 @@
+
+function kcAuthChecker() {
```

Comment:
There should be no need to put everything in a function to prevent stuff from leaking to the global scope once this file has been converted to a module. Which means that all the functions, variables, etc. could just be in the root context.